docs/package_templates/cmake_package/all/conanfile.py

```diff
@@ -89,8 +88,7 @@ def build_requirements(self):
         self.tool_requires("tool/x.y.z")
 
     def source(self):
-        get(self, **self.conan_data["sources"][self.version],
-                  destination=self.source_folder, strip_root=True)
```

Comment:
Where is the default destination? Does it get it from layout?

Reply:
It's just the current working directory by default. Documented [here](https://docs.conan.io/en/latest/reference/conanfile/tools/files/downloads.html#conan-tools-files-get). In the source method, the current working directory is set to `self.source_folder`.

Reply:
Revert, keep it to the source_folder. It's largely used in CCI and explicit makes it easier to understand the recipe.

Reply:
Done.